CHANGELOG.unreleased.md

```diff
@@ -11,6 +11,8 @@
 
 > Users must be able to say: “I had this issue, happy to know it's fixed”
 
+- [SR/XOSTOR] VG `linstor_group` removed if SR creation failed. (Deletion occurs only if `xo-server` was responsible for this VG creation) (PR [#7426](https://github.com/vatesfr/xen-orchestra/pull/7426))
```

Comment:
Is this understandable by users?

If no, change it, otherwise you can resolve this comment :slightly_smiling_face:

Reply:
I think it's understandable for users who faced this issue